src/plugins/vis_type_timeseries/public/application/components/index_pattern.js

```diff
@@ -229,6 +255,74 @@ export const IndexPattern = ({ fields, prefix, onChange, disabled, model: _model
           </EuiFormRow>
         </EuiFlexItem>
       </EuiFlexGroup>
+      {allowLevelofDetail && (
+        <EuiFlexGroup>
+          <EuiFlexItem>
+            <EuiFormRow
+              id={htmlId('detailLevel')}
+              label={
+                <>
+                  <FormattedMessage
+                    id="visTypeTimeseries.indexPattern.detailLevel"
+                    defaultMessage="Level of detail"
+                  />{' '}
+                  <EuiIconTip
+                    position="right"
+                    content={
+                      <FormattedMessage
+                        id="visTypeTimeseries.indexPattern.detailLevelHelpText"
+                        defaultMessage="Intervals will be selected automatically based on the current timeframe and configuration property {histogramMaxBars}."
+                        values={{ histogramMaxBars: UI_SETTINGS.HISTOGRAM_MAX_BARS }}
+                      />
+                    }
+                    type="questionInCircle"
+                  />
+                </>
+              }
+            >
+              <EuiFlexGroup responsive={false} alignItems="center">
+                <EuiFlexItem grow={false}>
+                  <EuiFormLabel>
+                    <FormattedMessage
+                      id="visTypeTimeseries.indexPattern.сoarse"
+                      defaultMessage="Coarse"
+                    />
+                  </EuiFormLabel>
+                </EuiFlexItem>
+                <EuiFlexItem grow={true}>
+                  <EuiRange
+                    id={htmlIdGenerator()()}
+                    value={model[maxBarsName]}
+                    onChange={handleMaxBarsChange}
+                    disabled={
+                      disabled ||
+                      isEntireTimeRangeActive(model, isTimeSeries) ||
+                      !(model[intervalName] === AUTO_INTERVAL || !model[intervalName])
+                    }
+                    min={0}
```

Comment:
This value is definitely wrong because it will never generate data. I think a value of 1 is probably too low as well. What do you think about 10?

Reply:
@wylieconlon I needed it earlier to correctly mark the scale from 0 to 100 with a step of 10.  When entering a number other than 0, the scale would be shifted. I handle that case here: 
![image](https://user-images.githubusercontent.com/20072247/100060255-60fce000-2e3d-11eb-8de8-363dc476f1d9.png)
Let's create a constant to make it a little obvious

Unfortunately even without a scale I cannot set` 1 `as a min value cause the `max - min` value must be a multiple of the `step` property, otherwise I see an error. Looks like a problem with EUI component. 
![image](https://user-images.githubusercontent.com/20072247/100059413-1a5ab600-2e3c-11eb-9042-46f6d7971341.png)

Also I'm not sure that 10 is a good default value for this property. I think for some cases 1 value can be useful and allows to emulate "Entire timerange mode" which we have on other tabs in TSVB.  Please keep in mind that to see data for that case you should set 
![image](https://user-images.githubusercontent.com/20072247/100061566-4b88b580-2e3f-11eb-98fc-f74ed5790a14.png)